src/Polysemy.hs

```diff
@@ -124,6 +131,14 @@ module Polysemy
   , (.@)
   , (.@@)
 
+    -- * 'RunH'
+    -- | When interpreting higher-order effects using 'interpretNew'
+    -- and friends, you can't execute higher-order "thunks" given by
```

Comment:
Not sure if this `"asdf"` isn't supposed to refer to a module `asdf`, according to haddock markup?

Reply:
Actually, from what I can tell haddock can tell `thunks` isn't a valid identifier for a module and won't try to convert it to a link.

... That said, I should escape it anyway.